taxi/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Driver, Manufacturer, Car
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (
+        ("Additional info", {"fields": ("license_number",)}),
+    )
+
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ("Additional info", {"fields": ("license_number",)}),
+    )
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_filter = [
+        "manufacturer",
+    ]
+    search_fields = [
+        "model",
+    ]
```

Comment:
Consider adding more fields to the `search_fields` attribute. It will provide a better search experience in the admin interface. For example, you could add 'license_plate' to the search fields if it exists in your Car model.